grails-app/services/de/iteratec/osm/measurement/environment/wptserver/LocationPersisterService.groovy

```diff
@@ -99,4 +103,36 @@ class LocationPersisterService implements iLocationListener {
             }
         }
     }
+
+    /**
+     * Parses the location label to determine the possible device type
+     * @param label location label for wptServer
+     */
+    DeviceType parseDeviceType(String label) {
```

Comment:
Suggestion: move this to `DeviceType` as a static method `guessDeviceType`, so others can easily profit from this logic.